tests/test.jl

```diff
@@ -1,35 +1,33 @@
-using Plots
+using PyPlot; plt = PyPlot
```

Comment:
PyPlot is imported in CMF.jl, so I don't think we should import it again.

Reply:
This is in the `tests` directory, not `src`. The file `test.jl` acts as if it were a user accessing `CMF`.

Reply:
ah right. okay cool.